### PR TITLE
add pathPrefix prompt when adding new server

### PR DESCRIPTION
**Pull request by yannip1234**
This PR fixes add an optional pathPrefix prompt when adding new server.

### CODE REVIEW

File: types/index.d.ts [hunk 1]
```diff
@@ -19,6 +19,7 @@ export interface IJSONServerSpec {
 	username?: string;
 	password?: string;
 	description?: string;
+	pathPrefix?: string;
```
Comment (gjsjohnmurray) [inline types/index.d.ts:22]: I don't think this is correct.

`pathPrefix` is a property of the `webServer` object and is already defined on the `IWebServerSpec` interface above.

File: src/api/addServer.ts [hunk 2]
```diff
@@ -59,6 +59,21 @@ export async function addServer(
...
 								spec.webServer.port = +portString;
+								const prefix = await vscode.window.showInputBox({
+									ignoreFocusOut: true,
+									placeHolder:
+										"Optional path prefix of instance",
+								});
+								if (typeof prefix !== "undefined") {
+									if (prefix) {
+										var pathPrefix = prefix.trim();
+										if (pathPrefix.charAt(0) !== '/') {
```
Comment (gjsjohnmurray) [inline src/api/addServer.ts:70]:
```suggestion
										if (pathPrefix.charAt(0) !== "/") {
```
Using doublequotes for consistency.

File: src/api/addServer.ts [hunk 3]
```diff
@@ -59,6 +59,21 @@ export async function addServer(
...
+											pathPrefix = '/' + pathPrefix;
```
Comment (gjsjohnmurray) [inline src/api/addServer.ts:71]:
```suggestion
											pathPrefix = "/" + pathPrefix;
```

**Review by gjsjohnmurray — CHANGES_REQUESTED** [review]
Thanks for the PR. I have suggested a few small changes.

**Comment by yannip1234** [comment]
Hi John, thanks for catching my error. I've implemented your suggestions!

**Review by gjsjohnmurray — APPROVED** [review]
:+1:

**Pull request merged**